processor/src/main/java/org/mobilitydata/gtfsvalidator/processor/TableLoaderGenerator.java

```diff
@@ -328,7 +329,7 @@ private MethodSpec generateLoadMethod() {
               gtfsTypeToParserMethod(field.type()),
               fieldColumnIndex(field.name()),
               RowParser.class,
-              field.required() ? "REQUIRED" : "OPTIONAL",
+              field.required() ? "REQUIRED" : (field.recommended() ? "RECOMMENDED" : "OPTIONAL"),
```

Comment:
Apologies for the nitpick, but I think this would be more straightforward if you referenced `FieldLevelEnum.class` directly here, instead of indirectly referencing the value via RowParser.  Aka replace the value on line 331.